gadgets/trace_open/gadget.yaml

```diff
@@ -38,3 +44,7 @@ params:
       key: failed
       defaultValue: "false"
       description: Show only failed events
+    paths:
+      key: paths
+      defaultValue: "false"
+      description: Show file path.
```

Comment:
Maybe also add "after symlink resolution" in here? I think it's more important here than in the field description.